cmd/rofl/mgmt.go

```diff
@@ -29,24 +33,150 @@ var (
 	scheme       string
 	adminAddress string
 
-	createCmd = &cobra.Command{
-		Use:   "create <policy.yml>",
-		Short: "Create a new ROFL application",
+	appTEE         string
+	appKind        string
+	deploymentName string
+
+	initCmd = &cobra.Command{
+		Use:   "init <name> [--tee TEE] [--kind KIND]",
```

Comment:
I would change behavior slightly:

```suggestion
		Use:   "init [PATH] [--tee TEE] [--kind KIND]",
```

So by default, it takes the current folder and creates rofl.yml with the project named after the folder name. If you provide `[PATH]`, then it creates the folder, names the project the same and uses it as a base folder for you rofl.yml.

Reply:
This has been implemented.

Reply:
Fixed the implementation so it behaves like `cargo init` et al. Please take another look.

Reply:
Yeah, works exactly as expected now. Thanks.